### PR TITLE
Fix part of #2833: Added missing named arguments in 'AdministratorControlsActivityTest' test file

### PR DESCRIPTION
**Pull request by dishantvyas15**
## Explanation

Fixes a part of the issue: #2833 Add missing named argument
Have added the missing named arguments in the 'AdministratorControlsActivityTest' test file.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -6,4 +6,4 @@
   <component name="ProjectType">
     <option name="id" value="Android" />
   </component>
-</project>
+</project>
```
Comment (prayutsu) [inline .idea/misc.xml]: @dishantvyas15 Please revert these changes as we don't need them.

File: app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt [hunk 2]
```diff
@@ -225,15 +225,15 @@ class AdministratorControlsActivityTest {
       onView(
         atPositionOnView(
           R.id.administrator_controls_list,
-          2,
+          position = 2,
           R.id.topic_update_on_wifi_switch
```
Comment (prayutsu) [inline app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt]: We want named arguments for all 4 parameters.

**Review by prayutsu — CHANGES_REQUESTED** [review]
@dishantvyas15 PTAL.

File: .idea/misc.xml (hunk 1 above)
Comment (dishantvyas15) [reply .idea/misc.xml]: @prayutsu, I have already made a commit (290ea32) to undo this change. Do I still need to do something else?

Comment (rt4914) [reply .idea/misc.xml]: EOF is missing, add one blank line at the end.

Comment (dishantvyas15) [reply .idea/misc.xml]: @rt4914 Done.

File: app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt (hunk 2 above)
Comment (dishantvyas15) [reply app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt]: Have made named arguments out of the parameters, as you mentioned,

**Review by rt4914 — APPROVED** [review]
LGTM, thanks. Will wait for @prayutsu review.

**Comment by rt4914** [comment]
Screenshot 2021-03-10 at 3 37 57 AM

This test is failing. @prayutsu can you check this?

**Comment by prayutsu** [comment]
@dipu989 LGTM, please resolve all the conversations initiated by me as they have been addressed.

**Comment by prayutsu** [comment]
> Screenshot 2021-03-10 at 3 37 57 AM
> 
> This test is failing. @prayutsu can you check this?

WIll look into this today and update the same.

**Review by prayutsu — APPROVED** [review]
@dishantvyas15 Please take the latest pull from the develop branch.

**Comment by dishantvyas15** [comment]
> @dishantvyas15 Please take the latest pull from the develop branch.

Have updated the fork. Glad to see "All checks have passed"!

**Comment by dishantvyas15** [comment]
Thanks, reviewers!
